sdk/rust/oak/src/lib.rs

```diff
@@ -425,9 +425,9 @@ pub fn set_panic_hook() {
 ///
 /// It has a single method for handling commands, which are [`Decodable`](crate::io::Decodable)
 /// objects that are received via the single incoming channel handle which is passed in at Node
-/// creation time. The return value is only used for logging in case of failure.
+/// creation time. The return value is only used for logging in case of error.
 pub trait CommandHandler<T: crate::io::Decodable> {
-    fn handle_command(&mut self, command: T) -> Result<(), crate::OakError>;
+    fn handle_command(&mut self, command: T) -> anyhow::Result<()>;
```

Comment:
Does `anyhow` propagate outside Wasm?

Reply:
Wasm does not know anything about anyhow or any other error, all the handling is done in our own code, which is what this PR implements.